README.md

```diff
@@ -9,3 +9,43 @@ Status and support
 - &#x2716; work in progress
 - &#x2714; supported
 - &#x2714; ongoing development
+
+### Usage
+
+Include the CDN in the desired file.
+```html
+<script type="text/javascript" src="https://cdn.jsdelivr.net/gh/c-hive/team-contribution-calendar/dist/team-contribution-calendar.min.js">
+</script>
+```
+
+#### Config
+
+<pre>
+<b>TeamContributionCalendar(container, gitHubUsers, gitLabUsers, proxyServerUrl)</b>
+</pre>
+    	
+Required params:
+- `container`: a DOM element in which the calendar will be rendered,
+- `gitHubUsers`: array of GitHub usernames(at least one).
```

Comment:
It would be enough to require 1 of either GH or GL, no?

Reply:
That's absolutely right.

I'll refine the related parts both in code and doc.